tests/image.js

```diff
@@ -87,11 +99,22 @@ describe( 'Image', () => {
 
 			// ContextualToolbar should not be visible.
 			expect( balloon.visibleView ).to.be.null;
+		} );
 
-			// Cleaning up.
-			editorElement.remove();
+		it( 'should listen to ContextualToolbar#show event with high priority', () => {
```

Comment:
Not sure if this test proves anything. It passes regardless of the priority of the listener in `Image.init()`, also on master. 

This assertion will always pass
```js
sinon.assert.notCalled( normalPrioritySpy );
```
because on master, the `Image.init()` listener is also attached with the `normal` priority and the order of `listenTo()` calls decides that the one in the test is always the last to execute. In fact – it never gets executed, because `Image` stops the event first.